tool/data/collector.js

```diff
@@ -24,16 +32,37 @@ async function getData(docID, sheetID, credentialsPath) {
     await doc.loadInfo();
     const sheet = doc.sheetsById[sheetID];
     const rows = await sheet.getRows();
-    const headerLength = rows[0]? rows[0]._sheet.headerValues.length: 0;
-    for (row of rows) {
-      const userData = row._rawData;
-      if (!rowValidator(userData, headerLength)) continue;
-      const uid = await sha256(userData[0]);
-      const role = userData[2];
+    for (remoteRow of rows) {
+      const userData = [...remoteRow._rawData];
+      const isRowValid = rowValidator(userData, USERS_COLUMN.EMAIL + 1);
+      const isOnePageTokenNullable = !remoteRow.one_page_token? true: false;
+
+      if (!isRowValid) continue;
```

Comment:
Not sure if it's suitable to log a warning here so somebody will go and fix the sheet.

Reply:
OK, I added this log. We can use these logs in troubleshooting.